src/lib/detection_algorithms/cntk/reviewer.js

```diff
@@ -0,0 +1,72 @@
+const path = require('path');
```

Comment:
Understood at the time i figured CNTK was using a standard format I see now that the format is specific to this code.